### PR TITLE
converted to GHA

### PR DESCRIPTION
**Pull request by johnricords**

### CODE REVIEW

**Comment by johnricords** [comment]
mockstack test is disable for release, module requires an array of vars

File: .travis.yml [hunk 1]
```diff
@@ -1,45 +0,0 @@
...
-    - stage: lint
-      name: Project Syntax Verification
-      script: make && make docker/run target=lint
-    - stage: lint
-      name: "CFN Version Validation"
-      install:
-        - pip install -r _requirements/cfn-version-check.txt
-        - yq --version
-      language: python
-      script: make cfn/version VERSION=$RELEASE_VERSION
```
Comment (lorengordon) [inline .travis.yml:23]: This project has a "custom" job that will require a new workflow, running `make cfn/version VERSION=$RELEASE_VERSION`... It's unique to this project, so I think just add a new workflow file that is triggered on both pull_request and workflow_call to run that command. Then update the release.yaml workflow to call that workflow as a reusable job, with `needs` on the release job. That should chain everything together well enough.

**Review by lorengordon — CHANGES_REQUESTED** [review]
See comment for custom workflow used by this project.

**Comment by lorengordon** [comment]
go codebuild go

**Pull request merged**